spec/lua/run_test_suite.lua

```diff
@@ -4,7 +4,7 @@ local luaunit = require("luaunit")
 local lfs = require("lfs")
 
 function srequire(t)
-    local success, err = xpcall(require, require_err_handler(t), t)
+    local success, err = xpcall(function() require(t) end, require_err_handler(t))
```

Comment:
I wonder if this is required to support versions older than Lua 5.3 like Lua 5.1, or it's just for readability?

Reply:
This is required for 5.1, xpcall in 5.1 does not take wrapped function arguments separately, and you must wrap it like that.
It's also forward compatible with 5.2 and 5.3 targets